store/search/src/main/scala/org/apache/spark/rpc/Master.scala

```diff
@@ -82,7 +80,7 @@ class Master(sparkConf: SparkConf) {
           do {
             try {
               LOG.info(s"starting registry-service on $hostAddress:$port")
-              val config = RpcEnvConfig(
+              val config = CarbonReflectionUtils.getRpcEnvConfig(
```

Comment:
Is it possible to move `CarbonReflectionUtils.getRpcEnvConfig` to this module so that this module doesn't need to depend on spark-common module?